Frontend/library/src/Inputs/MouseController.ts

```diff
@@ -253,56 +120,122 @@ export class MouseController {
         }
     }
 
-    /**
-     * Handles mouse enter
-     */
-    sendMouseEnter() {
-        if (!this.videoElementProvider.isVideoReady()) {
+    private sendMouseDown(button: number, X: number, Y: number) {
+        this.streamMessageController.toStreamerHandlers.get('MouseDown')?.([button, X, Y]);
+    }
+
+    private sendMouseUp(button: number, X: number, Y: number) {
+        const coord = this.coordinateConverter.normalizeAndQuantizeUnsigned(X, Y);
+        this.streamMessageController.toStreamerHandlers.get('MouseUp')?.([button, coord.x, coord.y]);
+    }
+}
+
+export class HoveringMouseController extends MouseController {
```

Comment:
HoverMouseController and MouseControllerHovering?

Is this a copy/paste error?